src/test/resources/features/RFMnfmNoIDFlow.feature

```diff
@@ -45,12 +45,11 @@ Feature: RFM CYA - NFM No ID flow
     When I refresh the page
     When I provide RFM New NFM Name as Test CYA
     Then I should navigate to RFM New NFM Contact Address Page
-    And I enter Country as United Kingdom
+    And I enter Country as United Arab Emirates
     And I click on Country selected
     And I click on Continue button
     Then I should see address error message Enter the first line of the address on the Address Line Element
```

Comment:
same here:
please move error validation tests to unit tests and remove from ATs

Reply:
This is the flow now So there wouldn't be United kingdom anymore so need to change it to different country.